syntax_checkers/go/govet.vim

```diff
@@ -23,7 +23,7 @@ function! SyntaxCheckers_go_govet_IsAvailable() dict
 endfunction
 
 function! SyntaxCheckers_go_govet_GetLocList() dict
-    let makeprg = self.getExecEscaped() . ' vet'
+    let makeprg = self.makeprgBuild({ 'args_after': 'vet' })
```

Comment:
could possibly just be `args` not `args_after`? Couldn't figure out, actually! Seems to work with just `args` though...